### DISCUSSION TITLE
사용법, 자주 묻는 질문들에 대한 답변은 Issues내의 메뉴에 있습니다.

### DISCUSSION THREAD

**Discussion by changdoc** [changdoc/naver-vaccine-macro#38] [Announcements]
자주 올라오는 질문은 가이드로 이슈들에 작성해두었습니다.

- 잠시 후 다시 시도하세요. (예약 신청 페이지 차단) 관련 #42 
- 최신 코드 혹은 특정 버전 사용 방법 #29
- 예약 신청 페이지 이동 목록 생성 방법 #11
- 병원 orgCd, sid 추출 및 예약 신청 페이지 url 추출 방법 #4 
- 네이버 맵에서 매크로 동작 방법 #165
- 카카오 맵에서 매크로 동작 방법 #166 

그 외 질문 답변은 이곳 Discussions 메뉴를 이용해주세요.

**Comment by hw0603** [comment]
안녕하세요, 지도 기반으로 코드를 새로 짜고 있는데, JSON에서 잔여백신이 발생하면 예약 페이지로 이동하는것 까지는 구현을 완료했습니다.
다만 프론트가 주가 아니다 보니.. 실제 잔여백신이 있을 때 HTML 구조가 어떻게 되는지 모르겠어서 진행을 못 하고 있습니다. 혹시 페이지 덤프 떠 놓으신 게 있으신가요?

**Reply by changdoc** [reply]
아니요 페이지 덤프는 따로 없고 제가 처음 작업할 당시에는 신청 페이지에 항상 잔여 수량이 0으로 표시가 되던 상황이였습니다.
이 프로젝트 contents.js 코드쪽을 보시면 query selector로 리스트 가져와서 백신 수량을 탐색하는 부분이 있어서 동일하게 사용하시면 될 듯하네요.

저도 지도 기반에서 새로고침 하는 부분을 작업 진행중이긴한데 진행하신것과 동작방식은 동일한것 같네요.

**Comment by dlwowlb** [comment]
여러 개의 창을 틀어서 여러 병원을 하면 차단 당하네요 현재로썬 한 개의 병원만 켜놔야 할 것 같습니다.

**Comment by changdoc** [comment]
추가로... 여기 댓글로 질문을 달아달라는 의도는 아니였어서;; 댓글을 막아두겠습니다.
